alias/gui.c

```diff
@@ -53,11 +53,33 @@ int alias_sort_name(const void *a, const void *b)
   const struct AliasView *av_a = a;
   const struct AliasView *av_b = b;
 
+  if (av_a->is_visible != av_b->is_visible) {
+    return av_a->is_visible ? -1 : 1;
+  }
+
```

Comment:
We're missing the 'both are invisible` check
```c
if (!av_a->is_visible)
  return 0;
```

Reply:
Unfortunately, adding that checks makes it impossible to do any kind of sorting without any active limit.

Reply:
Hmm... good point.
If we set `is_visible = true` in `menu_data_alias_add()` that should help.